node/src/command.rs

```diff
@@ -38,55 +38,61 @@ use {
     std::{io::Write, net::SocketAddr},
 };
 
-fn load_spec(id: &str, para_id: ParaId) -> std::result::Result<Box<dyn ChainSpec>, String> {
+fn load_spec(
+    id: &str,
+    para_id: Option<u32>,
+    container_chains: Option<Vec<String>>,
+    mock_container_chains: Option<Vec<u32>>,
+    invulnerables: Option<Vec<String>>,
+) -> std::result::Result<Box<dyn ChainSpec>, String> {
+    let para_id: ParaId = para_id.unwrap_or(1000).into();
+    let container_chains = container_chains.unwrap_or(vec![]);
```

Comment:
If the default value is going to be an empty vec, you can remove the `Option` from the argument, and have the caller pass `vec![]` instead of `None`.